static/app/components/stream/group.tsx

```diff
@@ -438,6 +439,14 @@ function BaseGroupRow({
     <GuideAnchor target="issue_stream" />
   );
 
+  const groupStats: ReadonlyArray<TimeseriesValue> = group.filtered
+    ? group.filtered.stats?.[statsPeriod]
+    : group.stats?.[statsPeriod];
+
+  const groupSecondaryStats: ReadonlyArray<TimeseriesValue> = group.filtered
+    ? group.stats?.[statsPeriod]
+    : [];
```

Comment:
This logic mirrors lines 36-43 in the old `groupChart.tsx`, and is one example of extracting the stats data in the parent component rather than in the groupChart component